libs/seiscomp/processing/amplitudes/iaspei.cpp

```diff
@@ -0,0 +1,168 @@
+#include <seiscomp/processing/amplitudes/iaspei.h>
+
+#include <vector>
+#include <complex>
+#include <cmath>
+
+namespace Seiscomp {
+
+namespace Processing {
+
+namespace IASPEI {
+
+
+static std::size_t
+find_absmax(std::size_t n, const double *f, double offset, std::size_t i1, std::size_t i2)
```

Comment:
This method is already available in `math/minmax.ipp`. Is it different to this one?